docs/tutorial/GetStarted.md

```diff
@@ -0,0 +1,387 @@
+
+Now, we can run the Universe server locally to test and install our package.
+
+
+#### Build the local Universe server
+Build the Universe Server Docker image
+```bash
+DOCKER_IMAGE="docker-user-name/universe-server" DOCKER_TAG="time-server" docker/server/build.bash
+```
+
+This will create a Docker image `universe-server:time-server` and `docker/server/target/marathon.json` on your local machine
+
+If you would like to publish the built Docker image, run
+```bash
+DOCKER_IMAGE="docker-user-name/universe-server" DOCKER_TAG="time-server" docker/server/build.bash publish
+```
+
+
+#### Run the local Universe server
+Using the `marathon.json` that is created when building Universe Server we can run a Universe Server in our DC/OS cluster which can then be used to install packages.
+
+Run the following commands inside the `server/target` directory to configure DC/OS to use the custom Universe Server (DC/OS 1.8+):
+
+`dcos marathon app add marathon.json`
+
+
+#### Add the Universe repo to DC/OS cluster:
+Now that you have local Universe server up and running, add this to the cluster instance. You can do this from the GUI or CLI. From the `server/target` directory execute
+
+`dcos package repo add --index=0 dev-universe http://universe.marathon.mesos:8085/repo`
+
+
+#### Install the package
+- You can search for you package using something like:
+
+    `dcos package search time`
+- Once you have found our `time-server` package, you can install it on to your cluster using
+
+    `dcos package install time-server`
+- Install the package and if everything works, you have successfully created a package, tested and deployed it!. You can check if your package is running at
+
+    `dcos marathon app list`
+
+- You can browse your endpoint by going to the cluster dashboard and clicking on Services > time-server, you will be able to see a current running task and you can click on any running task to view the endpoint url.
+
+Now continue to next step to publish your package to the DC/OS community.
+
+
+### Step 5 : Publish the package
+Universe Server is a new component introduced alongside `packagingVersion` `3.0`. In order for Universe to be able to provide packages for many versions of DC/OS at the same time, it is necessary for a server to be responsible for serving the correct set of packages to a cluster based on the cluster's version.
+
+All Pull Requests opened for Universe and the `version-3.x` branch will have their Docker image built and published to the DockerHub image [`mesosphere/universe-server`](https://hub.docker.com/r/mesosphere/universe-server/). In the artifacts tab of the build results you can find `docker/server/marathon.json` which can be used to run the Universe Server for testing in your DC/OS cluster.  For each Pull Request, click the details link of the "Universe Server Docker image" status report to view the build results.
```

Comment:
@ryadav88 we have many Universe builds, the one referred to in that PR *is* closed source, since it's under the "Mesosphere Only / Closed Source" project: [link](https://teamcity.mesosphere.io/viewType.html?buildTypeId=ClosedSource_Dcos_Universe_UniverseChecker_AutoCheckVersion3x).